Error: A `div` and `p` alone are insufficient for labeling a group of related fields. Use `fieldset` & `legend` or `role="group"` and `aria-labelledby` to provide the proper semantics.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/1/a126acd3-f8ad-4792-9c4f-1ed74b39e101.html b/output/Checkbox Group with Accessible Description/1/a126acd3-f8ad-4792-9c4f-1ed74b39e101.html
index 7ce04d3..46f9f90 100644
--- a/output/Checkbox Group with Accessible Description/1/a126acd3-f8ad-4792-9c4f-1ed74b39e101.html	
+++ b/output/Checkbox Group with Accessible Description/1/a126acd3-f8ad-4792-9c4f-1ed74b39e101.html	
@@ -1,7 +1,16 @@
-<div>
-  <p>What fruits do you like?</p>
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
-  <small>You can change this later</small>
-</div>
\ No newline at end of file
+<fieldset aria-describedby="fruit-desc">
+  <legend>What fruits do you like?</legend>
+  <small id="fruit-desc">You can change this later</small>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
+</fieldset>
\ No newline at end of file
